perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AuthzResolverMethod.java

```diff
@@ -782,5 +785,269 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			cz.metacentrum.perun.core.api.AuthzResolver.loadAuthorizationComponents(ac.getSession());
 			return null;
 		}
+	},
+
+	/*#
+	 * Get all Vos where the given user has set one of the given roles
+	 * or the given user is a member of an authorized group with such roles.
+	 *
+	 * @param user for who Vos are retrieved
+	 * @param roles for which Vos are retrieved
+	 * @return List of Vos
+	 *
+	 * @throws PrivilegeException when the principal is not authorized.
+	 */
+	/*#
+	 * Get all Vos where the given principal has set one of the given roles
+	 * or the given principal is a member of an authorized group with such roles.
+	 *
+	 * @param roles for which Vos are retrieved
```

Comment:
There is missing information about the type of parameter (List<String> for example here). The same problem is in the other Javadoc here.

Reply:
Fixed